app/src/main/java/org/ieeeguc/ieeeguc/models/Committee.java

```diff
@@ -0,0 +1,96 @@
+package org.ieeeguc.ieeeguc.models;
+
+import android.util.Log;
+
+import org.ieeeguc.ieeeguc.HTTPResponse;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.io.IOException;
+import java.util.ArrayList;
+
+import okhttp3.Call;
+import okhttp3.Callback;
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+import okhttp3.Response;
+
+/**
+ * Created by abdelrahmen on 16/02/17.
+ */
+
+public class Committee {
+
+    private String name;
+    private String description;
+    private ArrayList<User> members;
+    private int committeeId;
+
+    public Committee(String name,String description,ArrayList<User> members,int committeeId){
+
+        this.name = name;
+        this.description = description;
+        this.members = members;
+        this.committeeId = committeeId;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public String getDescription() {
+        return description;
+    }
+
+    public ArrayList<User> getMembers() {
+        return members;
+    }
+
+    public int getCommitteeId() {
+        return committeeId;
+    }
+
+    /**
+     * This method is called to get all the existing committees in the database.
+     * @param  {HTTPResponse} httpResponse [instance of HTTPResponse interface]
+     * @return {void}
+     */
+    public static void getAllCommittees(final HTTPResponse httpResponse){
```

Comment:
change the name of the variable to follow the conventions of the final variables naming